src/app/shared/directives/phone-validator/phone-validator.directive.ts

```diff
@@ -0,0 +1,15 @@
+import { Directive } from '@angular/core';
+import { NG_VALIDATORS, Validator, ValidationErrors, AbstractControl } from '@angular/forms';
+
+@Directive({
+  selector: '[lcPhoneValidator]',
+  providers: [{ provide: NG_VALIDATORS, useExisting: PhoneValidatorDirective, multi: true }],
+})
+export class PhoneValidatorDirective implements Validator {
+  validate(control: AbstractControl): ValidationErrors | null {
+    const PHONE_REGEXP = /[0-9]+/;
+    const isValid = PHONE_REGEXP.test(control.value);
+
+    return isValid ? null : { phoneValidator: { message: 'Номер телефона должен состоять исключительно из цифр' } };
+  }
+}
```

Comment:
I think we do not need this. I would create reactive form and set custom validator for such a purpose.

Reply:
done - replaced to reactive form